tests/functional/modules/test_zos_job_submit_func.py

```diff
@@ -446,9 +446,6 @@ def test_job_submit_pds_special_characters(ansible_zos_module):
         results = hosts.all.zos_data_set(
             name=data_set_name_special_chars, state="present", type="pds", replace=True
         )
-        hosts.all.shell(
-            cmd="echo {0} > {1}/SAMPLE".format(quote(JCL_FILE_CONTENTS), temp_path)
```

Comment:
Is this the line you were asking about being a duplicate?  It looks like the second echo call is being deleted here.

Reply:
Technically it was a triplicate as the merge was being managed by Git, 2 in dev and one in my merge, I limited it 1 and tested and regression passed.